MAINTAINERS-GUIDELINES.md

```diff
@@ -0,0 +1,73 @@
+
+Former [Reviewers](GOVERNANCE.md#reviewers) are removed from the [organization members](https://github.com/orgs/falcosecurity/people) if they are no longer listed in any [OWNERS](REPOSITORIES.md#owners) file.
+
+
+## Onboarding a Reviewer
+
+If [Community Members](GOVERNANCE.md#community-members) believe they match the criteria to become [Reviewers](GOVERNANCE.md#reviewers) of a repository (or a subdirectory) they can propose themselves by opening a PR to add themselves to the `reviewers` entry of the [OWNERS](REPOSITORIES.md#owners) of the repository (or the subdirectory). The person in question must publicly express their interest and should discuss it with the other persons listed in the OWNERS file and the community before proposing themself.
+
+[Maintainers](GOVERNANCE.md#maintainers) will review the PR and decide.
+
+If the decision is to grant the revier status, then the person in question must become a member of the [falcosecurity](https://github.com/falcosecurity) Github organization (see the [Organization membership](#organization-membership) section).
+
+## Onboarding a Maintainer
+
+If [Community Members](GOVERNANCE.md#community-members) believe they match the criteria to become [Maintainers](GOVERNANCE.md#maintainers) of a repository (or a subdirectory) they can propose themselves by opening a PR to add themselves to the `approvers` entry of the [OWNERS](REPOSITORIES.md#owners) of the repository (or the subdirectory). The person in question must publicly express their interest and should discuss it with the other persons listed in the OWNERS file and the community before proposing themself.
+
+[Maintainers](GOVERNANCE.md#maintainers) will review the PR and decide. Before taking the decision, existing maintainers may ask the person in question to shadow them or apply for a reviewer position for a period.
+
+If the decision is to grant the maintainer status, then the person in question must:
+- Become a member of the [falcosecurity](https://github.com/falcosecurity) Github organization, see the [Organization embership](#organization-membership) section.
+- Join with `*-maintainers` [GitHub team](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams) relative to the repository they became maintainer of (i.e. the [`falco-maintainers`](https://github.com/orgs/falcosecurity/teams/falco-maintainers) team for [falcosecurity/falco](https://github.com/falcosecurity/falco)); One can do so by opening a PR to change the [org.yaml](https://github.com/falcosecurity/test-infra/blob/master/config/org.yaml) file.
+- Update the [`persons.json`](https://github.com/falcosecurity/test-infra/blob/master/images/update-maintainers/persons.json) list by opening a PR to add their information.
+
+## Offboarding a Reviewer
+
+Reviewers of a repository (or a directory) can lose their status by voluntarily stepping down for personal reasons, or due to [inactivity](#review-maintainer-activity).
```

Comment:
might want to add the other common reasons here, such as violating the code of conduct

Reply:
Done